README.md

```diff
@@ -468,6 +469,252 @@ ANSWERS.addComponent('DirectAnswer', {
+
+```js
+{
+  type: "FIELD_VALUE",
+  answer: {
+    entityName: "Entity Name",
+    fieldName: "Phone Number",
+    fieldApiName: "mainPhone",
+    value: "+11234567890",
+    fieldType: "phone" 
+  },
+  relatedItem: { 
+    verticalConfigId: 'people',
+    data: { 
+      id: "Employee-2116",
+      type: "ce_person",
+      fieldValues: {
+        description: "This is the description field.",
+        name: "First Last",
+        firstName: "First",
+        lastName: "Last",
+        mainPhone: "+1234567890",
+      }
+    }
+  }
+}
+```
+
+A Direct Answer custom card needs a corresponding template. This can be added either inline by changing the component's constructor to:
+
+```js
+    constructor(config, systemConfig) {
+      super(config, systemConfig);
+      this.setTemplate(`<div> your template here </div>`)
+    }
+```
+
+Or by including a custom template bundle, and adding:
+
+```js
+  static defaultTemplateName () {
+    return 'CustomDirectAnswerTemplateName';
+  }
+```
+
+Where 'YourTemplateName' is the name of the template registered onto the Handlebars renderer.
+
+The example custom Direct Answer card will use the Handlebars template below.
+
+```hbs
+  <div class="customDirectAnswer">
+    <div class="customDirectAnswer-type">
+      {{type}}
+    </div>
+    <div class="customDirectAnswer-value">
+      {{#each customValue}}
+      {{#if url}}
+        {{> valueLink }}
+      {{else}}
+        {{{this}}}
+      {{/if}}
+      {{/each}}
+    </div>
+    {{> feedback}}
+  </div>
+
+  {{#*inline 'feedback'}}
+  <span class="customDirectAnswer-thumbsUpIcon js-customDirectAnswer-thumbsUpIcon"
+    data-component="IconComponent"
+    data-opts='{"iconName": "thumb"}'
+  ></span>
+  <span class="customDirectAnswer-thumbsDownIcon js-customDirectAnswer-thumbsDownIcon"
+    data-component="IconComponent"
+    data-opts='{"iconName": "thumb"}'
+  ></span>
+  {{/inline}}
+
+  {{#*inline 'valueLink'}}
+  <a class="customDirectAnswer-fieldValueLink" href="{{{url}}}"
+    {{#if @root/eventType}}data-eventtype="{{@root/eventType}}"{{/if}}
+    {{#if @root/eventOptions}}data-eventoptions='{{{ json @root/eventOptions }}}'{{/if}}>
+    {{{displayText}}}
+  </a>
+  {{/inline}}
+```
+
+Some css to flip the thumbs up icon the right way.
+
+```css
+  .customDirectAnswer-thumbsUpIcon svg {
+    transform: rotate(180deg);
+  }
+```
+
+This is an example of a possible custom Direct Answer card. It has custom rendering that depends on
+the fieldType of the directAnswer, as well as a custom analytics event.
+
+```js
+  class CustomDirectAnswerClassName extends ANSWERS.Component {
+    constructor(config, systemConfig) {
+      // If you need to override the constructor, make sure to call super(config, systemConfig) first.
+      super(config, systemConfig);
+
+      // For simplicity's sake, this custom Direct Answer card sets the template using setTemplate(), as opposed to
+      // a custom template bundle.
+      this.setTemplate(`
```

Comment:
I don't think you need to write out the entire template again here. Just do something like:

this.setTemplate(`TEMPLATE FROM ABOVE`)

There's already a lot going on with this code snippet.